appveyor.yml

```diff
@@ -96,7 +97,7 @@ test_script:
       $wc = New-Object 'System.Net.WebClient'
       $wc.UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\unittests.xml))
 
-      & codecov --no-color -X gcov -F unittests -e PYTHON_INSTALL
+      & $($env:PYTHON_INSTALL)\Scripts\codecov.exe --no-color -X gcov -F unittests -e PYTHON_INSTALL
```

Comment:
Probably it should looks like `${env:PYTHON_INSTALL}` instead `$($env:PYTHON_INSTALL)`
Also it should be changed at the line 117 
```
 & codecov --no-color -X gcov -F integrationtests_memcached -e PYTHON_INSTALL
```

Reply:
I'll try that (I took the syntax from the line right above, where the URL to upload the results to is constructed)!

Reply:
I also don't understand why it doesn't just pick up the `codecov` executable which is reachable in the` PATH` -- maybe the PowerShell script does not use the `PATH` the way I expect it to...

Reply:
Ok, I'm able to reproduce this error on local machine. Problem is related to codecov:
```
==> Preparing upload
    Error running `cd 'C:\projects\clcache' && git ls-files`: None
```
It try to `Popen("cd 'C:\projects\clcache' && git ls-files", stdout=PIPE, shell=True)` and it fails with 'The filename, directory name, or volume label syntax is incorrect.'
Here is the code:
```
try:
    from shlex import quote
except ImportError: # pragma: no cover
from pipes import quote
[...]
 root = quote(root)
[...]
 toc = str((try_to_run('cd %s && git ls-files' % root) or
[...]
```
quote's result is `"'C:\\projects\\clcache'"`